tests/Unit/NumericalAlgorithms/LinearOperators/Test_CoefficientTransforms.cpp

```diff
@@ -89,7 +89,8 @@ void check_transforms(
 
   // Back to nodal coefficients, which should match what we set up initially
   const NodalVectorType u_nodal = to_nodal_coefficients(u_modal, mesh);
-  CHECK_ITERABLE_APPROX(u_nodal_expected, u_nodal);
+  Approx local_approx = Approx::custom().epsilon(1e-12).scale(1.0);
```

Comment:
do we need the scale set to 1.0 or can we just do 1e-12?

Reply:
I haven't checked how `scale` works but I saw that other tests that use `CHECK_ITERABLE_CUSTOM_APPROX` use `scale(1.0)` (and the tests will fail to pass without `scale` regardless of tolerance)

`scale` can be left out when doing just `CHECK` tests though, so I think it's specific to the `ITERABLE` case.

Reply:
Oh, sorry, I should've clarified. `scale` sets the the absolute tolerance to `epsilon * scale` (if I've understood Catch documentation correctly), while `epsilon` sets a relative tolerance. In any case, leave it as-is.